extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/MSQWorkerTaskLauncher.java

```diff
@@ -345,6 +346,19 @@ private void runNewTasks()
     }
   }
 
+  /**
+   * Returns a pair which contains the number of currently running worker tasks and the number of worker tasks that are
+   * not yet fully started as left and right respectively.
+   */
+  public Pair<Integer, Integer> getWorkerTaskStatus()
```

Comment:
Code style question, would it be clearer if instead of a Pair, we create a simple class (in the task launcher itself) that encapsulates these numbers? I am fine either way since this is used only once, but it would be clearer to read if someone is just going through the ControllerImpl class.
PR LGTM apart from this. Thanks!